docs/Developers/Git_Basics.rst

```diff
@@ -70,7 +70,10 @@ Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
```

Comment:
Sort of agree.

I would like to see `git status` and `git remotes` explained in the previous section, with an explanation of `master`.

Then this section can be more forceful.  "Do not change files when you are on the master branch.  If you submit a pull request from your master branch, you are publicly stating that you do not understand git, and haven't read this document yet, and the coala developers may reject your work (even if it is good), because you dont yet know how to use the basic tools that enable collaboration on github."